src/features/login-form/index.tsx

```diff
@@ -26,16 +28,33 @@ export default function LoginForm() {
   const classes = useStyles();
   const [show, setShow] = useState(false);
 
-  const methods = useForm();
+  const methods = useForm<LoginSchema>({
+    resolver: zodResolver(loginSchema),
+  });
+  const {
+    handleSubmit,
+    formState: { errors },
+  } = methods;
+
+  const onSubmit = (data: LoginSchema) => {
+    return data;
+  };
 
   return (
     <FormProvider {...methods}>
-      <form>
-        <InputField type="text" placeholder="Email" contentBefore={<MailRegular />} name="email" />
+      <form onSubmit={(e) => void handleSubmit(onSubmit)(e)}>
+        <InputField
+          type="text"
+          placeholder="Email"
+          contentBefore={<MailRegular />}
+          name="email"
+          message={errors.email?.message ?? ''}
```

Comment:
```suggestion
          message={errors.email?.message}
```
if the message is not necessary, then everything is simplified